inspirehep/modules/records/mappings/records/journals.json

```diff
@@ -182,7 +182,13 @@
                             "copy_to": [
                                 "journalautocomplete"
                             ],
-                            "type": "string"
+                            "type": "string",
+                            "fields": {
+                              "raw": {
+                                "type": "string",
+                                "index": "not_analyzed"
+                              }
+                            }
```

Comment:
Why do you need to amend this mapping? You're not searching here. Or am I missing something?